src/components/Calculator.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="screen">
+        <input type="text" placeholder="0" readOnly />
+      </div>
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="box-color">÷</button>
+
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button" className="box-color">+</button>
+
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button" className="box-color">*</button>
+
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="box-color">-</button>
+
+      <button type="button" className="span-box">0</button>
+
+      <button type="button">.</button>
+      <button type="button" className="box-color">=</button>
+    </div>
```

Comment:
- [ ] It would be great if `Calculator.js` should have at least 1 child component to make your code reusable and make you understand how a react app deals with the components and it is required in the project requirements 👍